php/libraries/LorisForm.class.inc

```diff
@@ -108,18 +108,22 @@ class LorisForm
      * @param array  $attribs An array of other attributes that should
      *                        get added. Currently only the "class"
      *                        attribute gets added.
+     * @param bool   $getHTML Flag to return copy of $el.
      *
      * @return none, modifies $this->form as a side-effect.
      */
-    public function addSelect($name, $label, $options, $attribs = array())
-    {
+    public function addSelect($name, $label, $options, $attribs = array(),
+        $getHTML = null
+    ) {
```

Comment:
I think it would be better to get rid of the parameter and just always return the element. The parameters are supposed to be the same as QuickForm for compatability, the return isn't being used for anything else and can be ignored by the caller anyways, and it's better to have a well-defined return rather than one that switches based on a parameter.